lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,40 @@
+[GitLab Packages API](https://docs.gitlab.com/ee/api/packages.html) supports looking up package versions from [all types of packages registry supported by GitLab](https://docs.gitlab.com/ee/user/packages/package_registry/index.html) and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+To specify which specific repository should be queried when looking up a package, the `depName` should be formed with the project path first,
+then the `:` and finally the package name.
```

Comment:
```suggestion
To specify which specific repository should be queried when looking up a package, the `depName` should be formed with the project path first, then a `:` and finally the package name.
```

This can go on one line. 😉 
Also I like "then a `:`" better than "then the `:`".